FastMember.Signed.Tests/FastMember.Signed.Tests.csproj

```diff
@@ -8,8 +8,6 @@
     <AssemblyOriginatorKeyFile>../FastMember.snk</AssemblyOriginatorKeyFile>
     <SignAssembly>true</SignAssembly>
     <PublicSign Condition=" '$(OS)' != 'Windows_NT' ">true</PublicSign>
-    
-    <xUnitVersion>2.4.0-beta.2.build3981</xUnitVersion>
```

Comment:
Nothing remarkable here: the non-beta doesn't require extra CLI references, so this doesn't appear needed.